nima_core/dream.py

```
@@ -0,0 +1,76 @@
+                print(f"Dream consolidation completed successfully. Processed {memories_processed} memories.")
+            elif status == "no_memories":
+                print("Dream consolidation completed. No memories to process.")
+            elif status == "error":
+                error_msg = result.get("error", "Unknown error")
+                print(f"Dream consolidation failed: {error_msg}", file=sys.stderr)
+                return 1
+            else:
+                print(f"Dream consolidation finished with status: {status}")
+        
+        # Return 1 if status is error or None, 0 otherwise
+        return 0 if status != "error" else 1
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)